pkg/cmd/cli/cmd/project.go

```diff
@@ -227,6 +227,11 @@ func (o ProjectOptions) RunProject() error {
 		}
 	} else {
 		fmt.Fprintf(out, "Now using project %q on server %q.\n", namespaceInUse, clientCfg.Host)
+
+		// This means that we've just created a new project stanza.  take this opportunity to suggest oc status.
+		if contextNameIsGenerated {
+			fmt.Fprintln(out, "See more about your project using 'oc status'.")
```

Comment:
I don't think we need this section though - new-app is sufficient, and most of the time switching projects is something you're going to do after you already understand openshift enough to know about 'status'